### PR TITLE
Fix these @octokit/rest deprecations

### LINKED ISSUE #16: Fix these @octokit/rest deprecations
image

### PR DESCRIPTION
**Pull request by lavaldi**
Closes #16

### CODE REVIEW

File: src/utils/search.js [hunk 1]
```diff
@@ -1,10 +1,9 @@
 // This helper goes and fetches Good First Issues
 // This module expects a search query which is just a standard GitHub search query. These queries should not include sort or order, since those are defined in the function.
 const octokit = require("@octokit/rest")({
-  timeout: 0, // 0 means no request timeout
-  headers: {
-    accept: "application/vnd.github.v3+json",
-    "user-agent": "octokit/rest.js v1.2.3" // v1.2.3 will be current version
+  userAgent: "octokit/rest.js v1.2.3",
```
Comment (ashimasingla20) [inline src/utils/search.js:4]: Hi i am new to open source can you tell me from where you figured out headers needs to be removed and user-agent needs to be replaced by userAgent?

Comment (lavaldi) [reply src/utils/search.js:4]: Hi @ashimasingla20 this change is according to the documentation https://octokit.github.io/rest.js/#usageand if you see the issue #16 the warning message recommends do this

Comment (ashimasingla20) [reply src/utils/search.js:4]: Got it, Thanks.

**Comment by siddhant1** [comment]
Works awesome, Thanks

**Pull request merged**